src/test/java/service/AmountHandlerBalanceTest.java

```diff
@@ -0,0 +1,13 @@
+package service;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class AmountHandlerBalanceTest {
+
+    @Test
+    public void getAmount() {
+        AmountHandlerBalance amountHandlerBalance = new AmountHandlerBalance();
+        Assert.assertEquals("",(int)10,(int)amountHandlerBalance.getAmount(10));
```

Comment:
```suggestion
        Assert.assertEquals("", 10, amountHandlerBalance.getAmount(10));
```
the same for other similar cases